packages/backend/src/house/house.util.ts

```diff
@@ -1,17 +1,11 @@
 // Generate random house code
-// Inspired by:
-// https://nodejs.org/api/crypto.html#cryptorandombytessize-callback
-
 import { Injectable } from '@nestjs/common';
+import { randomInt } from 'crypto';
 
 @Injectable()
 export class HouseUtil {
   generateString(length) {
     const max_safe = 281474976710655;
```

Comment:
an attribution + comments might be good here.
what is the significance of this number?